src/getEntryFields.js

```diff
@@ -1,5 +1,8 @@
-module.exports = entry => {
+module.exports = (entry, resolve = null) => {
```

Comment:
remove resolve entry stuff from pull request
```suggestion
module.exports = entry => {
```